docs/installation.rst

```diff
@@ -67,27 +62,38 @@ Execute the following command to install the latest development build of Transfo
 
 .. code-block:: bash
 
-  # Execute one of the following commands
-  NVTE_FRAMEWORK=pytorch pip install git+https://github.com/NVIDIA/TransformerEngine.git@main    # Build TE for PyTorch only. The default.
-  NVTE_FRAMEWORK=jax pip install git+https://github.com/NVIDIA/TransformerEngine.git@main        # Build TE for JAX only.
-  NVTE_FRAMEWORK=tensorflow pip install git+https://github.com/NVIDIA/TransformerEngine.git@main # Build TE for TensorFlow only.
-  NVTE_FRAMEWORK=all pip install git+https://github.com/NVIDIA/TransformerEngine.git@main        # Build TE for all supported frameworks.
-  
+  pip install git+https://github.com/NVIDIA/TransformerEngine.git@main
+
+This will automatically detect any installed deep learning frameworks and build support for them. Set `NVTE_FRAMEWORK` in the environment (as a comma-separated list) to explicitly specify which frameworks to support.
```

Comment:
```suggestion
This will automatically detect supported deep learning frameworks installed and build TransformerEngine for them. Set `NVTE_FRAMEWORK` in the environment (as a comma-separated list) to explicitly specify which frameworks to support.
```